app/helpers/react_components/donations/footer_form.rb

```diff
@@ -12,6 +12,8 @@ def to_s
               }
             },
             user_signed_in: user_signed_in?,
+            captcha_required: !current_user&.github_auth?,
+            recaptcha_site_key: '6LfFYEUjAAAAAH9eRl1qeO2R9aXzdXGnAybe6ulM', # TODO: use secret
```

Comment:
@iHiD We'll probably want to store this as a secret (I'll deprecate this key)

Reply:
Yep - you ok to do it?

Reply:
Sure